packages/beeq/src/components/card/scss/bq-card.scss

```diff
@@ -0,0 +1,25 @@
+/* -------------------------------------------------------------------------- */
+/*                        Card styles                        */
+/* -------------------------------------------------------------------------- */
+
+@import './bq-card.variables';
+
+:host {
+  @apply block;
+}
+
+.bq-card {
+  @apply relative flex rounded-[--bq-card--borderRadius] border-[length:--bq-card--borderWidth] border-[color:--bq-card--borderColor] bg-[--bq-card--background];
+}
+
+/** 
+ * Set the padding based on the type selected
+*/
+
+.bq-card__default {
+  @apply p-[--bq-card--padding];
+}
+
+.bq-card__minimal {
+  @apply p-0;
+}
```

Comment:
Maybe we could move all these rules to JSX, they aren't so many, what do you think?